gradle/dependencies.gradle

```diff
@@ -74,7 +74,7 @@ ext.deps = [
         indexing: "com.jetbrains.intellij.platform:indexing-impl:${versions.idea}",
         asm: "org.jetbrains.intellij.deps:asm-all:7.0.1",
     ],
-    sqlitePsi: "com.alecstrong:sqlite-psi-core:0.3.15",
+    sqlitePsi: "app.softwork:sqlite-psi-core:0.3.16-SNAPSHOT",
```

Comment:
I backported sql-psi too and will upload a non snapshot version after successful build. Alternative alecstrong could update his version